### ISSUE TITLE
I have too many agenda items

### ISSUE THREAD

**Issue by littledan** [tc39/agendas#271]
I don't want to crowd out other people from talking, or take over topics that other want to champion. However, I want to help some important topics move forward. For the balance, maybe a couple things could be adjusted in the agenda, but I am wondering about your thoughts:
- I could deprioritize my topics in the agenda (within the category) to give everyone else a chance to speak first
- I can mention that I'd be happy to have others presenting these topics if they want to (but I am not sure when in the meeting that would be appropriate).

Any other ideas? We have a general issue whhere there are a lot of smart people in the room who don't seem to be empowered to champion proposals.

**Comment by snuggs** [comment]
@littledan I run NYC.rb and is a meetup group of about 5,000 members. Whenever we are obtaining consensus during organizer meetings we often limit questions to one per person (amongst other conventions). This usually cuts down at least 40-50% of deliberation time.

Also have read some interesting protocols in [Robert's Rules of Order](https://en.wikipedia.org/wiki/Robert%27s_Rules_of_Order)

TL;DR; _Even if an organization has adopted Robert's Rules of Order, it can still adopt its own rules which supersede any rules in this book._

YMMV of course and take it as a grain of salt. At minimum guaranteed to help break through a cognitive stalemate as an organizer.

**Comment by jridgewell** [comment]
@snuggs: Not quite a perfect fit. @littledan _presents_ several (too many apparently) proposals during the meetings, he doesn't hog up discussion time after the presentation.

**Comment by bakkot** [comment]
@littledan I don't personally have any problem with you presenting on a bunch of different topics. It's better than them being neglected!

That said, I'd be happy to present the subclassing-private-static item you have on the agenda for this meeting, especially if you don't mind me just using the slides you've already made.

**Comment by snuggs** [comment]
@jridgewell Ah i see. Just sounds like a delegation situation then. In that case I'd say if I can't get at least one person that can cheerlead the proposal I probably am going to have a difficult time gaining interest in the first place. Is there anyone that can champion some?

**Comment by snuggs** [comment]
Oh right! What @bakkot said... Github doesn't seem to update comments as frequently as you'd think. #HardRefreshLife :-/

**Comment by littledan** [comment]
@bakkot Sounds good. I gave you edit access to the slides and wrote your name next to the agenda item. Thanks for your help!

**Comment by littledan** [comment]
@snuggs Interesting idea!

**Comment by keithamus** [comment]
Can this be closed now @littledan has moved their topics to the end?

**Comment by littledan** [comment]
I have too many agenda items for this meeting too. As things stand, I'll be presenting for roughly one of the three days. If anyone wants to take on co-championing and/or presenting any of these items, I'd love to work with you, please just let me know. Also, if you have any feedback about which of these items might be lower priority/make sense to not present, that would also be helpful. Once more people have added agenda items, I'll do the same as at the last meeting and move my items to the bottom within each timebox.
